doc/configuration.md

```diff
@@ -96,6 +97,16 @@ itself. It defaults to `false`.
 verbose_trace: true
 ```
 
+### `chain_stack_traces`
+
+This boolean field controls whether or not stack traces are chained. 
+Disabling stack trace chaining will improve performance at the cost of
```

Comment:
"improve performance for heavily asynchronous code". I don't want people disabling stack chains in synchronous code for an imagined perf gain :stuck_out_tongue:.